metrics/exporter.go

```diff
@@ -126,11 +131,11 @@ func newMetricsExporter(config *metricsConfig, logger *zap.SugaredLogger) (view.
 	// If there is a Prometheus Exporter server running, stop it.
 	resetCurPromSrv()
 
-	if ce != nil {
-		// UnregisterExporter is idempotent and it can be called multiple times for the same exporter
-		// without side effects.
-		view.UnregisterExporter(ce)
+	// TODO: Move Stackdriver and Promethus operations before stoping to an interface.
```

Comment:
[nit] stopping/stoppable should have two "p"s

Reply:
Done.